servo:
  [
    { id: 1, speed: 700, acc: 0 },
    { id: 2, speed: 50, acc: 30 },
    { id: 3, speed: 2400, acc: 80 },
    { id: 4, speed: 2400, acc: 80 },
    { id: 5, speed: 300, acc: 0 },
    { id: 6, speed: 35, acc: 20 },
  ]

#x, y, z
default_action: [0.0, 10.2264, 51.9209]

back_action: [0.0, -20.4149, 36.4906]
  
down_action: [-32.0955, -13.315, 5.85853]

put1_action: [-36.9788, -27.4871, 4.73369]

put2_action: [-36.8733, -11.2739, 4.73369]

put3_action: [-36.7788, 3.6604, 4.73369]

